Lib/fontTools/designspaceLib/__init__.py

```diff
@@ -1412,3 +2878,51 @@ def loadSourceFonts(self, opener, **kwargs):
                 loaded[source.path] = source.font
             fonts.append(source.font)
         return fonts
+
+    @property
+    def formatTuple(self):
+        """Return the formatVersion as a tuple of (major, minor).
+
+        .. versionadded:: 5.0
+        """
+        if self.formatVersion is None:
+            return (5, 0)
+        numbers = (int(i) for i in self.formatVersion.split("."))
+        major = next(numbers)
+        minor = next(numbers, 0)
+        return (major, minor)
+
+    def buildStatTable(self, ttFont: TTFont) -> None:
```

Comment:
@anthrotype says: remove this method or move it to `varLib` + same for the `stat` and `statNames` modules